treeherder/jobs/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class JobDetail(models.Model):
+    '''
+    Represents metadata associated with a job
+    '''
+    id = models.AutoField(primary_key=True)
+    job_guid = models.CharField(max_length=50, db_index=True)
+    title = models.CharField(max_length=200, null=True)
```

Comment:
In the UI html, it looks like it expects to always have a ``title`` and ``value`` but that ``url`` can be null.  Should the title not have the ``null=True`` here?